.gitignore

```diff
@@ -1,12 +1,6 @@
 # If a change is made in here, add it to .npmignore as well unless you
 # explicitly want to include it in the published artifact
-node_modules/*
-!node_modules/@okta
-node_modules/@okta/*
-!node_modules/@okta/i18n
-!node_modules/@okta/courage
-!node_modules/@okta/jquery.simplemodal
-!node_modules/@okta/qtip2
+node_modules
```

Comment:
I see, so the ``packages`` directory won't contain extra stuff like the past ``node_modules`` directory did?